src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Reflection;
```

Comment:
this file should have a copyright header similar to the other .cs files in this project.

Reply:
Added header